pkg/core/state/notification_event.go

```diff
@@ -20,6 +20,79 @@ type NotificationEvent struct {
 	Item       *stackitem.Array `json:"state"`
 }
 
+// ContractInvocation contains method call information.
+// The Arguments field will be nil if serialization of the arguments exceeds a predefined limit
+// (for security reasons). In that case Truncated will be set to true.
+type ContractInvocation struct {
```

Comment:
Move it to a separate go file. `notification_event.go` is for `NotificationEvent` structure.